test/HealthChecks.CosmosDb.Tests/HealthChecks.CosmosDb.approved.txt

```diff
@@ -3,27 +3,44 @@ namespace HealthChecks.CosmosDb
     public class CosmosDbHealthCheck : Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheck
     {
         public CosmosDbHealthCheck(string connectionString) { }
+        public CosmosDbHealthCheck(Microsoft.Azure.Cosmos.CosmosClient cosmosClient, HealthChecks.CosmosDb.CosmosDbHealthCheckOptions options) { }
         public CosmosDbHealthCheck(string connectionString, string database) { }
         public CosmosDbHealthCheck(string accountEndpoint, Azure.Core.TokenCredential tokenCredential, string database) { }
         public CosmosDbHealthCheck(string connectionString, string? database, System.Collections.Generic.IEnumerable<string>? containers) { }
         public CosmosDbHealthCheck(string accountEndpoint, Azure.Core.TokenCredential tokenCredential, string? database, System.Collections.Generic.IEnumerable<string>? containers) { }
         public System.Threading.Tasks.Task<Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckResult> CheckHealthAsync(Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckContext context, System.Threading.CancellationToken cancellationToken = default) { }
     }
+    public sealed class CosmosDbHealthCheckOptions
+    {
+        public CosmosDbHealthCheckOptions() { }
+        public System.Collections.Generic.IEnumerable<string>? ContainerIds { get; set; }
+        public string? DatabaseId { get; set; }
+    }
     public class TableServiceHealthCheck : Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheck
     {
-        public TableServiceHealthCheck(string connectionString, string tableName) { }
```

Comment:
Following the pattern of the other Azure Storage health checks, I changed the attribute of the `string` to be nullable, so that users may optionally check a given table. This isn't a binary-breaking change, as this is only checked at compile-time, but it may cause code to no longer compile upon update if they use a nullable context.

Reply:
ok